localstack/aws/app.py

```diff
@@ -31,6 +32,7 @@ def __init__(self, service_manager: ServiceManager = None) -> None:
                 handlers.add_internal_request_params,
                 handlers.handle_runtime_shutdown,
                 metric_collector.create_metric_handler_item,
+                load_service_for_data_plane,
```

Comment:
@dfangl handling that case would likely need custom logic for the distributed LocalStack scenario because service detection happens in the proxy, right?

Reply:
that's a great point actually! curious to get daniel's perspective

Reply:
Sorry for the late reply, I actually had one here, but forgot to submit the review, it still shows as pending :(
You can route based on the `lambda-url` in the domain, so it should work without a problem.